macos/xcode/.swiftlint.yml

```diff
@@ -0,0 +1,193 @@
+included: # paths to include during linting. `--path` is ignored if present.
```

Comment:
Do we plan to share the same ios and macos yml file eventually?

Reply:
Yes. That makes sense. I moved it to the root of the folder and now the lint path is being passed through an environment variable.

Reply:
Nice! :D